apps/api/src/lib/i18n-adapter.ts

```diff
@@ -11,11 +11,8 @@ export default class I18nAdapter {
       lng: DEFAULT_LOCALE,
       ns: ['emails'],
       backend: {
-        loadPath: path.join(__dirname, '../../locales/{{lng}}/{{ns}}.json'),
-        addPath: path.join(
-          __dirname,
-          '../../locales/{{lng}}/{{ns}}.missing.json'
-        ),
+        loadPath: path.join(__dirname, 'locales/{{lng}}/{{ns}}.json'),
+        addPath: path.join(__dirname, 'locales/{{lng}}/{{ns}}.missing.json'),
```

Comment:
Due to how Nx bundles the code, the `__dirname` will refer to the root output directory. It's also configured to do a simple copy of the `apps/api/src/locales` directory.